public/flightControl/index.html

```diff
@@ -1,525 +1,40 @@
-	data = JSON.parse(data);
-
-	switch(data.type) {
-		case 'status':
-			handler = processStatusMessage;
-			break;
-		case 'list_reply':
-			handler = processListReply;
-			break;
-		case 'load_reply':
-			handler = processLoadReply;
-			break;
-		default:
-			console.log('Unknown message type - ' + data.type);
-			return;
-	}
-	handler(data);
-});
-
-// Repeatable Functions
-
-swapControlPanels = function(mode) {
-	// Already in the right view, no-op
-	if (model.view === mode) {
-		return;
-	}
-	model.view = mode;
-	$('.preFlight').slideToggle();
-	$('.flightReady').slideToggle();
-	enable($('#playDrones'));
-	disable($('#pauseDrones'));
-	disable($('#stopDrones'));
-};
-
-function setValue(item, value) {
-	if(value) {
-		setTrue(item);
-	} else {
-		setFalse(item);
-	}
-}
-
-function setTrue(item) {
-	item.removeClass('false tralse').addClass('true');
-}
-
-function setTralse(item) {
-	item.removeClass('false true').addClass('tralse');
-}
-
-function setFalse(item) {
-	item.removeClass('true tralse').addClass('false');	
-}
-
-function disable(item) {
-	item.prop('disabled', 'true');
-}
-
-function enable(item) {
-	item.removeAttr('disabled');
-}
-
-function gridSetup() {
-	var halfWidth = canvasWidth / 2;
-	var halfHeight = canvasHeight / 2;
-
-	for (var x = 0.5; x < canvasWidth; x += 10) {
-		context.moveTo(x, 0);
-		context.lineTo(x, canvasHeight);
-	}
-
-	for (var y = 0.5; y < canvasHeight; y += 10) {
-		context.moveTo(0, y);
-		context.lineTo(canvasWidth, y);
-	}
-
-	context.strokeStyle = '#eee';
-	context.stroke();
-	
-	context.beginPath();
-	context.moveTo(halfWidth, 0);
-	context.lineTo(halfWidth, canvasHeight);
-
-	context.moveTo(0, halfHeight);
-	context.lineTo(canvasWidth, halfHeight);
-
-	context.strokeStyle = '#000';
-	context.stroke();
-}
-
-drawDrone = function(position, index) {
-	// TODO: Actually draw drones
-	$("#droneRawCoordintes").append("Drone #" + Number(index + 1) + ": [" +  position.x +  ", " +  position.y + "]<br>");
-}
-
-</script>
-</body>
-</html>
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta http-equiv='Content-Type' content='text/html; charset=UTF-8'>
+		<meta name='mobile-web-app-capable' content='yes'>
+		<meta name='apple-mobile-web-app-capable' content='yes'>
+		<meta name='apple-mobile-web-app-status-bar-style' content='black'>
+		<meta name='viewport' content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui'>
+		<meta name="theme-color" content="#0f9d58">
+		<title>Drone Controller</title>
+
+		<!-- Custom Styles -->
+		<link rel="stylesheet" href="css/styles.css">
+
+		<!-- Style Dependecies -->
+		<link async rel="stylesheet" href="/node_modules/roboto-fontface/css/roboto/roboto-fontface.css">
+		<link async rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
+		<link async rel="stylesheet" href="node_modules/material-design-lite/material.min.css">
+		<link async rel="stylesheet" href="node_modules/bootstrap-material-design/dist/css/ripples.min.css">
```

Comment:
@dmz38 I was getting a failure in loading styles related to this:
<img width="741" alt="screen shot 2017-03-04 at 11 26 31 pm" src="https://cloud.githubusercontent.com/assets/22605445/23584671/d6b06198-0135-11e7-8afb-f392badd7918.png">

Not really breaking anything, I can look at fixing it later.